Add Ruby 2.0.0 to list of supported rubies

sevos opened the pull request.

sevos committed:
Add Ruby 2.0.0 to list of supported rubies

sferik commented:
If it's going to be supported, it needs to be [tested](https://github.com/rubygems/gemcutter/blob/master/.travis.yml).

sevos committed:
Add Ruby 2.0.0 to .travis.yml

sevos committed:
Add require 'rubygems/command' in order to fix tests

Tests were not passing on my machine until I made those changes.
Rubygems 2.0.3.

sevos commented:
Should be fine now :smile:
